templates/Dataland/v2/index.html.twig

```diff
@@ -19,7 +19,7 @@
 {% block body %}
     <div class="bootstrap">
         <div class="container">
-            <div class="row">
+            <div class="row pb-2">
```

Comment:
This fixed it, but in my mind is a bit tight. Should be consider `pb-4`?

Reply:
You are right, i changed it to increase the padding